Chapter1/Tests/CalculatorTests/DivisionTests.swift

```diff
@@ -0,0 +1,22 @@
+import XCTest
+@testable import Calculator
+
+extension CalculatorTests {
+  func testDivision_whenDividingBySameNumber_returnsOne() throws {
+    let first = 5
+    let second = 5
+    
+    let result = try Calculator.division(first, second)
+    
+    XCTAssertEqual(result, 1)
+  }
+  
+  func testDivision_whenDividingByZero_returnsNil() throws {
+    let first = 3
+    let second = 0
+    
+    let result = try? Calculator.division(first, second)
```

Comment:
This is not the best approach. We are not checking that an error is raised when we are dividing by zero. Also, we do not test which kind of error is thrown.

This test will work well also if the `division` signature would be 
```swift
func division(_ num: Int, _ den: Int) -> Int?
```
which is not.

When a function declares that it `throws`, we want to test that an error is thrown in those conditions and which actual error is thrown.

You can use the [XCTAssertThrowsError](https://developer.apple.com/documentation/xctest/error_assertions) function to verify that.